exercises/react/example/Reactor.ceylon

```diff
@@ -0,0 +1,88 @@
+import ceylon.collection { HashMap, MutableMap }
+
+alias Element => Integer;
+
+class Reactor() {
```

Comment:
in *this* example solution, the reactor is absolutely useless; it does nothing. But I wanted to leave it in case a student's implementation truly needs it (for example, they store all their cells in an array and iterate over the array to propagate values)